packages/http-specs/spec-summary.md

```diff
@@ -7127,6 +7127,178 @@ Expect to handle a unknown type value. Mock api will return 'test'
 
 Expect to send a string value. Mock api expect to receive 'test'
 
+### Type_Union_Discriminated_Envelope_Object_CustomProperties_getCustomProperties
+
+- Endpoint: `get /type/union/discriminated/envelope/object/custom-properties`
+
+Test discriminated union with custom property names.
+When value of query parameter "petType" is "cat" or no query parameter input, the expected response is:
+
+```json
+{
+  "petType": "cat",
+  "petData": {
+    "name": "Whiskers",
+    "meow": true
+  }
+}
+```
+
+When it is "dog", expected response is:
+
+```json
+{
+  "petType": "dog",
+  "petData": {
+    "name": "Rex",
+    "bark": false
+  }
+}
+```
+
+### Type_Union_Discriminated_Envelope_Object_CustomProperties_putCustomProperties
+
+- Endpoint: `put /type/union/discriminated/envelope/object/custom-properties`
+
+Test discriminated union with custom property names.
+Send the union as:
+
+```json
+{
+  "petType": "cat",
+  "petData": {
+    "name": "Whiskers",
+    "meow": true
+  }
+}
+```
+
+### Type_Union_Discriminated_Envelope_Object_Default_getDefault
+
+- Endpoint: `get /type/union/discriminated/envelope/object/default`
+
+Test discriminated union with envelope serialization.
+When value of query parameter "kind" is "cat" or no query parameter input, the expected response is:
+
+```json
+{
+  "kind": "cat",
+  "value": {
+    "name": "Whiskers",
+    "meow": true
+  }
+}
+```
+
+When it is "dog", expected response is:
+
+```json
+{
+  "kind": "dog",
+  "value": {
+    "name": "Rex",
+    "bark": false
+  }
+}
+```
+
+### Type_Union_Discriminated_Envelope_Object_Default_putDefault
```

Comment:
for all of these test cases, we don't need to reiterate . For example, update `Default_getDefault` to just `Default_get`, `Default_putDefault` to `Default_put`, `Type_Union_Discriminated_Envelope_Object_CustomProperties_getCustomProperties` to `Type_Union_Discriminated_Envelope_Object_CustomProperties_get`